src/app/static/src/app/components/projectListPage.vue

```diff
@@ -118,24 +144,48 @@
             },
             createProject() {
                 const regionNames = this.regions.map((tag) => tag.text.trim());
-                if (regionNames.length == 0) {
-                    // user has only entered one region name and has not blurred the input
-                    // so this.regions is empty even though this.newRegion is populated
-                    // so take this.newRegion as the only region
-                    regionNames.push(this.newRegion)
+                const newRegion = this.newRegion.trim();
+                if (newRegion.length > 0) {
```

Comment:
Have also changed this so it always adds the last typed region (once validated) even if the user hasn't blurred the input and created a tag.

Reply:
Good idea, that's caught me out before!